### DISCUSSION TITLE
It is easy to insert something, how about remove something?

### DISCUSSION THREAD

**Discussion by PannenetsF** [chhoumann/quickadd#427] [Q&A]
We can easily insert some date via QuickAdd, but if we want to remove the added thing, what should we do?

It comes from my checkbox: when I toggle a checkbox as done, QA would insert a `[done: {{DATE}}]` for me. But if it is mistakes, what should I do to remove the tag?

**Comment by chhoumann** [comment]
Hey @PannenetsF!

So, do you use macros to do this, or a single capture? I think I might be able to support reversing captures & templates, but macros are almost impossible to support.

In general, if I mistakenly insert something in my active file, I just CTRL+Z it away.
